frontend/components/home/hero.tsx

```diff
@@ -16,11 +16,11 @@ function Hero(): JSX.Element {
 							{t('home.hero.body')}
 						</p>
 
-						<div className="mx-auto mt-5 max-w-md sm:flex sm:justify-center md:mt-8">
-							<LinkButtonLG href="/create-review">
+						<div className="mx-auto mt-5 max-w-md sm:flex sm:justify-center md:mt-8" >
+							<LinkButtonLG href="/create-review" umami="homepage Submit a Review button"  >
```

Comment:
Believe this needs to be data-umami-event

Reply:
umami is just the interface property name. It pulls through to the html as data-umami-event. I am working on fixing the errors caught by test-fe right now.